{"name": "test_17_form_authentication_wrong_password", "status": "passed", "steps": [{"name": "Открытие главной страницы сайта", "status": "passed", "start": 1664807534883, "stop": 1664807536903}, {"name": "Открытие страницы с примером Form Authentication", "status": "passed", "start": 1664807536903, "stop": 1664807537469}, {"name": "Заполнение формы username", "status": "passed", "start": 1664807537469, "stop": 1664807537557}, {"name": "Заполнение формы password с ошибкой", "status": "passed", "start": 1664807537558, "stop": 1664807537637}, {"name": "Нажатие кнопки Login", "status": "passed", "start": 1664807537637, "stop": 1664807538038}], "attachments": [{"name": "stdout", "source": "d513c8a2-7b61-4033-ad6a-5b4294446c5b-attachment.txt", "type": "text/plain"}], "start": 1664807534883, "stop": 1664807538062, "uuid": "bbb1d645-20e2-4f50-9efc-2101de7ba3e2", "historyId": "18e236a33097a3cbc0abb97b1b33780d", "testCaseId": "6de87b31751ffac9d334d19d5734daf8", "fullName": "pom.tests.test_diplom#test_17_form_authentication_wrong_password", "labels": [{"name": "suite", "value": "Авторизации с ошибкой в password"}, {"name": "feature", "value": "Проверка авторизации c ошибкой в password"}, {"name": "tag", "value": "@pytest.mark.usefixtures('text_inside_test')"}, {"name": "parentSuite", "value": "pom.tests"}, {"name": "host", "value": "DESKTOP-I5N6LU0"}, {"name": "thread", "value": "45600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pom.tests.test_diplom"}]}